src/display.js

```diff
@@ -0,0 +1,61 @@
+function celsiustoFahrenheit(degree) {
```

Comment:
- [ ] Let's make sure to convert all the function to arrow function syntax in this file.